README.md

```diff
@@ -10,6 +10,17 @@ Debezium is a distributed platform that turns your existing databases into event
 
 Debezium is open source under the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0.html)
 
+# Purpose of the Debezium UI
+The Debezium UI provides standalone web application, which connects to Kafka Connect via its REST API. See the [Debezium UI docs](https://debezium.io/documentation/reference/operations/debezium-ui.html) for more information
+
+# Debezium environment vars
```

Comment:
Could you add a small sentence explaining the table? Saying that it shows the ENV vars for the Debezium UI container image (with Docker Hub link) and it's related application.properties parameter name when running the java application locally.

Reply:
```suggestion
# Configure the Debezium UI
```